+++
title = "Programme"
description = "Programme"
+++


## Lecture and Lab topics

- RNA-Seq from raw data to DESeq2: mapping, quantification at gene and transcript level, gamma-Poisson model, DESeq2
- Single cell (droplet based) RNA-seq: exploratory analysis, quality assessment, dimension reduction (incl., t-SNE, UMAP) 
- Visualisation / graphics, PCA and other low-dimensional embeddings, Clustering, distances, nearest-neighbour graphs (with sc-RNA-Seq examples)
- Statistical hypothesis testing, false discovery rate, multiple testing, filtering and weighting
- Regression: more on design matrices, power, identifiability, diagnostics, generalized linear models for count data
- Classification / supervised machine learning
- Image-based data and Spatial omics (CODEX, MERFISH et al.)
- Introduction to the Bioconductor project (community, organisational structure, website, important infrastructure packages, data structures, annotation resources) <!-- Lori Kern -->
- Immunobioinformatics  <!-- Katharina Imkeller -->
- Mass spectrometry-based proteomics and metabolomics <!-- Laurent Gatto, Johannes Rainer --> 
- Emerging topics and participant suggestions

<!--
<div class="row">
<div class="col-sm-offset-1 col-sm-10">
-->

## Sunday, 11 June

<table class="programme">
	<tbody>
		<tr>
			<td width="100ex">18:00–20:00</td>
			<td>Registration &amp; Installation help desk</td>
		</tr>
		<tr>
			<td>18:00–20:00</td>
			<td>Get Together with drinks and nibbles</td>
		</tr>
	</tbody>
</table>

<table class="programme" width="850px">
	<tbody>
	<tr>
Please see <a href="https://github.com/Bioconductor/CSAMA">here for the teaching material - lectures and labs</a>.
    </tr>
    </tbody>
</table>
	
## Monday, 12 June

<table class="programme" width="850px">
	<tbody>
		<tr>
			<td width="100ex">08:30–09:15</td>
			<td width="90ex"><strong>Lecture 01</strong></td>
			<td>Introduction to <a href="https://www.bioconductor.org">Bioconductor</a> (Lori Kern)</td>
		</tr>
		<tr>
			<td>09:15–10:00</td>
			<td><strong>Lecture 02</strong></td>
			<td>PCA and other low-dimensional embeddings (Robert Gentleman)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>10:00–10:30</td>
			<td colspan="2"><em>Coffee</em></td>
		</tr>
		<tr>
			<td>10:30–11:15</td>
			<td><strong>Lecture 03</strong></td>
			<td><a href="https://www.huber.embl.de/msmb/Chap-Testing.html">Statistical tests</a> (Wolfgang Huber)</td>
		</tr>
		<tr>
			<td>11:15–12:00</td>
			<td><strong>Lecture 04</strong></td>
			<td>Design of High Throughput Experiments and their Analysis (Charlotte Soneson)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>12:00–14:00</td>
			<td colspan="2"><em>Lunch break (Self sourced in Brixen)</em></td>
		</tr>
		<tr>
			<td>13:30–14:00</td>
			<td>&nbsp;</td>
			<td>Installation help desk</td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab</strong></td>
			<td>R and Bioconductor Basics</td>
		</tr>
		<tr>
			<td>17:00–17:30</td>
			<td><strong>Flashlight talks</strong></td>
			<td>
				<ul>
				  <li>Julia Philipp</li>
					<li>Sara Lobato Moreno</li>
					<li>Jamie Yam Auxillos</li>
					<li>Caroline Lohoff</li>
					<li>Dawn Shuiping Lin </li>
				</ul>
				Each talk 5 min + 2 min questions
			</td>
		</tr>
		<tr>
			<td>20:10–22:00</td>
			<td><strong>Evening session</strong></td>
			<td>Drinks will be provided</td>
		</tr>
		<tr>
			<td>20:30–21:15</td>
			<td><strong>Group work | Demos</strong></td>
			<td>
			<ul>
              <li>Group projects (Vincent Carey, Robert Gentleman)
 			  <li>Interactive exploration and data quality assessment for matrix-shaped data with <a href="https://doi.org/doi:10.18129/B9.bioc.iSEE">iSEE</a> (Charlotte Soneson)
 			  <li>Tidy analysis of genomic data with <a href="https://sa-lee.github.io/plyranges/">plyranges</a> and other packages (Michael Love)
              <li>HDF5 / large single-cell datasets (Davide Risso)
			</ul>
			</td>
		</tr>
		<tr>
			<td>21:15–22:00</td>
			<td><strong>Follow-up discussions on the lawn</strong></td>
			<td><ul>
			</ul></td>
		</tr>
	</tbody>
</table>

## Tuesday, 13 June

<table class="programme" width="850px">
	<tbody>
		<tr>
			<td width="100ex">08:30–09:15</td>
			<td width="90ex"><strong>Lecture 05</strong></td>
			<td>RNA-Seq intro: biology of transcription, quantification, batch effects and QC (Michael Love)</td>
		</tr>
		<tr>
			<td>09:15–10:00</td>
			<td><strong>Lecture 06</strong></td>
			<td>RNA-Seq for DE: Types of DE, modeling counts, scaling, transcript lengths, parameter estimation (Charlotte Soneson)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>10:00–10:30</td>
			<td colspan="2"><em>Coffee</em></td>
		</tr>
		<tr>
			<td>10:30–11:15</td>
			<td><strong>Lecture 07</strong></td>
			<td>Single-cell RNA-seq: exploration, quality control, low-dimensional embeddings (Davide Risso)</td>
		</tr>
		<tr>
			<td>11:15–12:00</td>
			<td><strong>Lecture 08</strong></td>
			<td>Distances, nearest-neighbour graphs and clustering (Vincent Carey)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>12:00–14:00</td>
			<td colspan="2"><em>Lunch break (Self sourced in Brixen)</em></td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab 3</strong></td>
			<td>End-to-end RNA-Seq workflow (Michael Love and Charlotte Soneson)</td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab 4</strong></td>
			<td>Single-cell transcriptomics (Davide Risso)</td>
		</tr>
		<tr>
			<td>17:00–17:30</td>
			<td><strong>Flashlight talks</strong></td>
			<td>
				<ul>
					<li>Giulia Graziano</li>
					<li>Karin Prummel</li>
					<li>James Bryson</li>
					<li>Alina Batzilla</li>
				</ul>
				Each talk 5 min + 2 min questions
			</td>
		</tr>
		<tr>
			<td>20:10–22:00</td>
			<td><strong>Evening session</strong></td>
			<td>Drinks will be provided</td>
		</tr>
		<tr>
			<td>20:30–21:15</td>
			<td><strong>Demos / tutorials</strong></td>
			<td>
			<ul>
			  <li>What you need to know for submitting a package to Bioconductor (Lori Kern)
			  <li>Reproducible research and open science (Laurent Gatto, Charlotte Soneson)
		      <li>Shiny (Vincent Carey) 
	        </ul>
			</td>
		</tr>
		<tr>
			<td>21:15–22:00</td>
			<td><strong>Follow-up discussions on the lawn</strong></td>
			<td><ul><!--
			  <li>... <selected from participants' suggestions> ...
			  <li>Spatial omics (Wolfgang)
			  <li>Publishing computational research (Vince, Robert)-->
			</ul></td>
		</tr>
	</tbody>
</table>

## Wednesday, 14 June

<table class="programme">
	<tbody>
		<tr>
			<td width="100ex">08:30–09:15</td>
			<td width="90ex"><strong>Lecture 09</strong></td>
			<td>Human Cell Atlas |> Ontologies |> Shiny %<>% Bioconductor (Vincent Carey)</td>
		</tr>
		<tr>
			<td>09:15–10:00</td>
			<td><strong>Lecture 10</strong></td>
			<td>Spatial (transcript)omics (Davide Risso)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>10:00–10:30</td>
			<td colspan="2"><em>Coffee</em></td>
		</tr>
		<tr>
			<td>10:30–11:15</td>
			<td><strong>Lecture 11</strong></td>
			<td>Advanced transcriptomic inference: bootstrapping (transcript-level inferential uncertainty), pseudo-bulking, double-dipping (Michael Love and Charlotte Soneson)</td>
		</tr>
		<tr>
			<td>11:15–12:00</td>
			<td><strong>Lecture 12</strong></td>
			<td>Bioconductor Annotation Resources | Caching (Lori Kern)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>12:00–14:00</td>
			<td colspan="2"><em>Lunch break (Self sourced in Brixen)</em></td>
		</tr>
		<tr>
			<td>14:00–22:30</td>
			<td><strong>Social programme</strong></td>
			<td><a href="{{< relref "about.md#social" >}}">Excursion to the mountains and dinner</a></td>
		</tr>
	</tbody>
</table>

## Thursday, 15 June

<table class="programme">
	<tbody>
		<tr>
			<td width="100ex">08:30–09:15</td>
			<td width="90ex"><strong>Lecture 13</strong></td>
			<td>Mass spectrometry-based proteomics, incl. single-cell (Laurent Gatto)</td>
		</tr>
		<tr>
			<td>09:15–10:00</td>
			<td><strong>Lecture 14</strong></td>
			<td>Mass spectrometry-based metabolomics (Johannes Rainer)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>10:00–10:30</td>
			<td colspan="2"><em>Coffee</em></td>
		</tr>
		<tr>
			<td>10:30–11:30</td>
			<td><strong>Lecture 15</strong></td>
			<td>Regression (Sarah Kaspar)</td>
		</tr>
		<tr>
			<td>11:30–12:00</td>
			<td><strong>Lecture 16</strong></td>
			<td><a href="https://www.huber.embl.de/msmb/Chap-Graphics.html">Visualization</a> (Wolfgang Huber)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>12:00–14:00</td>
			<td colspan="2"><em>Lunch break (Self sourced in Brixen)</em></td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab 5</strong></td>
			<td>Proteomics/metabolomics (Laurent Gatto, Johannes Rainer)</td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab 6</strong></td>
			<td>Multiple testing and FDR (Wolfgang Huber)</td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab 7</strong></td>
			<td><a href="https://nullranges.github.io/tidy-ranges-tutorial/">Tidy genomic analysis workflows</a> (Michael Love)</td>
		</tr>
		<tr>
			<td>14:00–17:00</td>
			<td><strong>Lab 9</strong></td>
			<td>Analysis of untargeted metabolomics data (Johannes Rainer)</td>
		</tr>
		<tr>
			<td>17:00–17:30</td>
			<td><strong>Flashlight talks</strong></td>
			<td>
				<ul>
					<li>Felina Lenkeit</li>
					<li>Anna Diamant</li>
					<li>Mathilde Moens</li>
					<li>Gaël Hammer</li>
				</ul>
			</td>
		</tr>
	</tbody>
</table>

## Friday, 16 June

<table class="programme">
	<tbody>
		<tr>
			<td width="100ex">08:30–09:15</td>
			<td width="90ex"><strong>Lecture 17</strong></td>
			<td>Machine learning (Robert Gentleman)</td> 
		</tr>
		<tr>
			<td>09:15–10:00</td>
			<td><strong>Lecture 18</strong></td>
			<td>Immunoinformatics (Katharina Imkeller)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>10:00–10:30</td>
			<td colspan="2"><em>Coffee</em></td>
		</tr>
		<tr>
			<td>10:30–11:15</td>
			<td><strong>Lecture 19</strong></td>
			<td>Advanced single cell omics II: Trajectories, pseudo-time, bifurcations, spatial statistics (Charlotte | Davide | Wolfgang)</td>
		</tr>
		<tr>
			<td>11:15–12:00</td>
			<td><strong>Lecture 20</strong></td>
			<td>Group projects (Robert Gentleman, Vincent Carey)</td>
		</tr>
		<tr bgcolor="#eeeeee">
			<td>12:00–14:00</td>
			<td colspan="2"><em>Lunch break</em></td>
		</tr>
		<tr>
			<td>14:00–16:30</td>
			<td><strong>Lab 5</strong></td>
			<td>Proteomics/metabolomics (Laurent Gatto, Johannes Rainer)</td>
		</tr>
		<tr>
			<td>14:00–16:30</td>
			<td><strong>Lab 6</strong></td>
			<td>Multiple testing and FDR (Wolfgang Huber)</td>
		</tr>
		<tr>
			<td>14:00–16:30</td>
			<td><strong>Lab 7</strong></td>
			<td><a href="https://nullranges.github.io/tidy-ranges-tutorial/">Tidy genomic analysis workflows</a> (Michael Love)</td>
		</tr>
		<tr>
			<td>14:00–16:30</td>
			<td><strong>Lab 9</strong></td>
			<td>Analysis of untargeted metabolomics data (Johannes Rainer)</td>
		</tr>
		<tr>
			<td>16:30-17:00</td>
			<td><strong>Q&A</strong></td>
			<td>Career panel (open questions for instructors)</td>
		</tr>
        <tr>
			<td>17:00</td>
			<td>&nbsp;</td>
			<td>Closing remarks</td>
		</tr>
	</tbody>
</table>


### Lecturers and Teaching Assistants

{{< lecturersList >}}

<!--
</div>
</div>
-->
